### PR TITLE
Added env variables for max image and video size.

### PR DESCRIPTION
**Pull request by LadyLumb**
I changed the IMAGE_LIMIT and VIDEO_LIMIT variables to read from the env, or use default if not set in the env, to make it easier to change this on an instance by instance basis and make it persistent across updates.

### CODE REVIEW

**Comment by LadyLumb** [comment]
Sidenote, the env variable must be entered in bytes to work.

So 'MAX_IMAGE_SIZE=40000' would be 40 megabytes.

**Comment by ThibG** [comment]
Can you document it in `.env.production.sample`?

**Comment by LadyLumb** [comment]
Done as commit 30d82fc

**Comment by ThibG** [comment]
Unfortunately, the commit hasn't been added to the PR itself. Also, 1MB = 1024*1024 bytes, not 1000 bytes ^^'

**Comment by LadyLumb** [comment]
Yeah imma level with you I got no idea what I'm doing haha. I think it should be merged properly now and I fixed the nerd numbers uwu

File: .env.production.sample [hunk 1]
```diff
@@ -160,6 +160,12 @@ STREAMING_CLUSTER_NUM=1
 # Maximum number of pinned posts
 # MAX_PINNED_TOOTS=5
 
+# Maximum image and video upload sizes
+# Units are in bytes
+# 1024 bytes equals 1 megabyte
```
Comment (ThibG) [inline .env.production.sample]: 1024 bytes equals 1 kilobyte though

Comment (LadyLumb) [reply .env.production.sample]: Oh damn that's my bad lol I know this, just wasnt thinking. Is fixed now.

**Pull request merged**